ts/utils/strings.ts

```diff
@@ -30,19 +30,24 @@ export function isTextIncludedCaseInsensitive(
  * @param text
  * @param separator
  */
-export function capitalize(text: string, separator: string = " ") {
-  return text
+export function capitalize(text: string, separator: string = " "): string {
```

Comment:
Instead of manually handling the capitalization logic, what do you think about using Lodash's `upperFirst` method? It could simplify the code and make it a bit more concise. Let me know what you think!